docs/modules/about/pages/about.adoc

```diff
@@ -54,16 +54,15 @@ You could build your images and move the logic from the scripts executed via `in
 The scripts configure only a certain number of things.
 Each deployment would likely need additional configuration, which is the main reason for using a Persistent Volume Claim (PVC) for the configuration directory of the Core OpenNMS instance.
 
-We must place the core configuration on a PVC configured as `ReadWriteMany` to allow the use of independent UI servers so that the Core can make changes and the UI instances can read from them.
+We must place the core configuration on a PVC configured as `ReadWriteMany` to allow the use of independent UI servers so that the Core can make changes.
```

Comment:
I think we can safely remove this line as we are not using `ReadWriteMany` and we have dropped support for UI servers.

Lets confirm with @deejgregor, to make sure it makes sense to drop this line.